src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -2164,6 +2173,12 @@ static void uct_ib_mlx5dv_check_dm_ksm_reg(uct_ib_mlx5_md_t *md)
                  ucs_status_string(status));
         return;
     }
+
+    ucs_debug("%s: KSM over device memory is supported",
+              uct_ib_device_name(&md->super.dev));
+
+#else
+    ucs_debug("UCX was compiled without --with-dm configuration");
```

Comment:
IMO this debug log is not needed as we don't usually have such messages, and we can know the compilation defines from `ucx_info -b`